### DISCUSSION TITLE
Create release notes for previous artifact version

### DISCUSSION THREAD

**Discussion by charlieliu2001** [rfennell/AzurePipelines#1501] [Q&A] [answered]
Good day @rfennell ,

I'm trying to create release notes for a specific artifact version (which is older) in Azure using the _Generate Release Notes based on Release Comparison API_ extension. However, the release notes would always contain the latest changes (Pull requests, commits, work items, etc.). I tried enabling _Show Primary Only_ under _Advanced_ settings but that didn't change anything in my case. I also tried disabling _Find Associated PRs_.

Is there a way to create release notes for a older artifact, and to ignore everything that came after it?

For example, the latest version is 20230419.1, but I want to create release notes for 20221205.9. When trying to creating release notes, I get release notes that includes information up until the latest version (20230419.1) whereas I only want information up until the specified version (20221205.9).

![image](https://user-images.githubusercontent.com/43513073/233110211-872772ed-c185-447a-bdaf-97148bb3e940.png)

Thanks,
Charlie

**Comment by rfennell** [comment, marked as answer]
No, there is no way to do what you have described with my extension. All of it's output is driven for the current release.
